cluster-sync/ember/loop_back.yaml

```diff
@@ -0,0 +1,73 @@
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: ember-csi-lvm
+---
+apiVersion: v1
+kind: PersistentVolumeClaim
+metadata:
+  name: loop-back-pvc
+  namespace: ember-csi-lvm
+  labels:
+    app: loop-back-lvm
+spec:
+  storageClassName: local
+  accessModes:
+     - ReadWriteOnce
+  resources:
+    requests:
+      storage: 25Gi
+---
+apiVersion: apps/v1
+kind: Deployment
```

Comment:
Would a Job/Pod be better?  As-is, deleting the deployment pod puts it in permanent Error/CrashloopBackoff state.

Reply:
Yeah deleting the pod will mess with all kinds of things on the host. So it doesn't really make sense to do a deployment, might as well be a pod.